src/backend/users/views.py

```diff
@@ -55,6 +55,16 @@ def get(self, request, **kwargs):
         return response
 
 
+class DeleteToken(APIView):
+    permission_classes = (IsAuthenticated,)
+
+    def get(self, request, **kwargs):
+        response = HttpResponseRedirect("/")
```

Comment:
I don't think the redirect is necessary here--we can simply return a `200` status of some kind. It was more important on logging in because the initial request was coming from a user clicking an email link, so the Javascript part of the app hadn't loaded yet. When we're logging out, we don't have that problem, so we can just rely on the frontend to navigate back to `/` (and in fact, if we leave this out it still redirects back to `/` correctly).